vformer/models/classification/convvt.py

```diff
@@ -0,0 +1,127 @@
+import torch
+import torch.nn as nn
+from timm.models.layers import trunc_normal_
+
+from ...encoder.convvt import ConvVTStage
+from ...utils import MODEL_REGISTRY
+
+
+@MODEL_REGISTRY.register()
+class ConvVT(nn.Module):
+    """
+    Implementation of CvT: Introducing Convolutions to Vision Transformers:
+    https://arxiv.org/pdf/2103.15808.pdf
+
+    Parameters:
+    ------------
+    img_size: int
+        Size of the image, default is 224
+    in_channels:int
+        Number of input channels in image, default is 3
+    num_stages: int
+        Number of stages in encoder block, default is 3
+    num_classes: int
+        Number of classes for classification, default is 1000
+    * The following are all in list of int/float with length num_stages
+    patch_size: [int]
```

Comment:
```suggestion
    patch_size: list[int]
            Size of patch, default is [7, 3, 3]
    patch_stride: list[int]
```

we are using docstrings in documentation [website](https://vformer.readthedocs.io/en/latest/) so I think it is better to write this way, let me know your thoughts on this

Reply:
Yeah, would be good to write the docstrings in this way since they are being used to generate docs.

Reply:
Fixed here 4a252e0